spec/presenters/tree_builder_alert_profile_obj_spec.rb

```diff
@@ -0,0 +1,126 @@
+describe TreeBuilderAlertProfileObj do
+  before do
+    role = MiqUserRole.find_by(:name => "EvmRole-operator")
+    group = FactoryGirl.create(:miq_group, :miq_user_role => role, :description => "Tags Group")
+    login_as FactoryGirl.create(:user, :userid => 'tags_wilma', :miq_groups => [group])
+  end
+
```

Comment:
`let!` is force-evaluated before a test runs and `let` only when its needed. We should try to use the minimal number of `let!`s in a code. 

For example this:
```ruby
let!(:a) { 1 }
let!(:b) { 2 }
let!(:c) { [a,b] }
```

Should be written as:
```ruby
let(:a) { 1 }
let(:b) { 2 }
let!(:c) { [a,b] }
```
Assuming that `c` is required in a test BEFORE it gets actually referenced in the code...